doc/releases/release-notes-3.2.rst

```diff
@@ -293,7 +293,6 @@ Drivers and Sensors
   * The Zephyr network CAN bus driver, which provides raw L2 access to the CAN bus via a CAN
     controller driver, has been moved to :zephyr_file:`drivers/net/canbus.c` and can now be enabled
     using :kconfig:option:`CONFIG_NET_CANBUS`.
-  * STM32: Now supports dual CAN instances.
```

Comment:
Why is this deleted

Reply:
It's in the commit log and the description: "removed duplicate entry".